rkt/image/fetcher.go

```diff
@@ -77,6 +79,13 @@ func (f *Fetcher) fetchImageDeps(hash string) error {
 		}
 		f.addImageDeps(hash, imgsl, seen)
 	}
+
+	if common.SupportsOverlay() {
```

Comment:
@iaguis So now if the image has no dependency, seems it will not render tree store?